gencert-private.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
```

Comment:
In order to avoid repetitions and errors, I would suggest to use variables to refer names of out files (as in `gencert-public.sh`)
https://github.com/italia/spid-compliant-certificates/blob/d709afe6e94e548d53691b9bf9b40283057c73c7/gencert-public.sh#L3-L5